doc/index.rst

```diff
@@ -1,10 +1,55 @@
-The Plutus Platform
+Plutus Tools SDK User Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
-To read more about the Platform, see :ref:`what_is_the_plutus_platform`.
-To get started using the Platform see :ref:`plutus_getting_started`.
+Intended Audience
+-------------------------------
 
+This user guide is intended for developers who are authoring distributed applications 
+("DApps") by using smart contracts on the Cardano blockchain. 
+
+.. note::
+   If you are a developer who wants to contribute to the Plutus Tools SDK project, 
+   please refer to documentation residing in the `Plutus Tools SDK repository <https://github.com/
+   input-output-hk/plutus-apps>`_.
+
+Plutus Tools SDK
```

Comment:
I think the Trace Emulator is worth highlighting here as a tool. It's used for testing plutus contracts on an emulated blockchain. https://github.com/input-output-hk/plutus-apps/blob/main/plutus-contract/src/Plutus/Trace/Emulator.hs